trade-finance-manager-api/src/v1/content-strings/index.js

```diff
@@ -106,6 +106,57 @@ const DEAL = {
           text: 'Any upfront or arrangement fees, as well as interest rates and risk margin fees are in an approved currency.',
         },
       },
+      2: {
```

Comment:
this version should be in main already

Reply:
Cheers - when I put the PR up it wasn't in main and was causing some issues - merged main ino there, it's gone now